cypress/integration/jsPromisesPracticeDOM.spec.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-Cypress.Commands.add('clickButton', (mouseButton) =>
-  cy.get('body').trigger('mousedown', mouseButton).trigger('mouseup')
-);
-
-Cypress.Commands.add('checkMessageDisplayed', (message) =>
-  cy.get('[data-qa="notification"]').contains(message)
-);
+const page = {
+  notification: () => cy.get('[data-qa=notification]'),
+  leftBtnClick: () => {
+    cy.get('body').trigger('mousedown', { button: 0 }).trigger('mouseup');
+  },
+  rightBtnClick: () => {
+    cy.get('body').trigger('mousedown', { button: 2 }).trigger('mouseup');
+  },
```

Comment:
Достатньо знайти елемент
```suggestion
  body: () => cy.get('body),
```
і далі викликати де треба 
```js
page.body.click();
page.body.rightclick();
```